### PR TITLE
Bug 1949202: Add metadata for ppc64le

### PR DESCRIPTION
**Pull request by manojnkumar**
Add metadata to indicate availability of sriov-network-operator on the ppc64le architecture.

### CODE REVIEW

File: manifests/4.8/sriov-network-operator.v4.8.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -371,6 +371,8 @@ spec:
   labels:
     olm-owner-enterprise-app: sriov-network-operator
     olm-status-descriptors: sriov-network-operator.v4.8.0
+    operatorframework.io/arch.amd64: supported
```
Comment (zshi-redhat) [inline manifests/4.8/sriov-network-operator.v4.8.0.clusterserviceversion.yaml:374]: There are several operands managed by SR-IOV Operator, do you think we need to label them with the same tags?(I'm not very famiilar with how multi-arch works, any reference would be helpful here).

**Review by zshi-redhat — COMMENTED** [review]
@manojnkumar  thanks for the PR!
sriov operator has an [upstream project](https://github.com/k8snetworkplumbingwg/sriov-network-operator), would you mind sending the patch there, then we will backport to downstream once merged upstream.

btw, is this required for 4.8 FF?

**Comment by manojnkumar** [comment]
> @manojnkumar thanks for the PR!
> sriov operator has an [upstream project](https://github.com/k8snetworkplumbingwg/sriov-network-operator), would you mind sending the patch there, then we will backport to downstream once merged upstream.
> 
> btw, is this required for 4.8 FF?

@zshi-redhat : I do not see a 4.8 directory upstream? Should I be creating that?  Not required for 4.8 FF, afaik.

**Comment by zshi-redhat** [comment]
> > @manojnkumar thanks for the PR!
> > sriov operator has an [upstream project](https://github.com/k8snetworkplumbingwg/sriov-network-operator), would you mind sending the patch there, then we will backport to downstream once merged upstream.
> > btw, is this required for 4.8 FF?
> 
> @zshi-redhat : I do not see a 4.8 directory upstream? Should I be creating that? Not required for 4.8 FF, afaik.

Looking at the upstream project, I think we had stopped updating the openshfit manifest there. My bad, we should not be proposing a PR to upstream.

**Comment by zshi-redhat** [comment]
@manojnkumar what's the implication or making this change? 
I'm not ware of any sriov function testing on ppc64le or amd64, how it is verified?

**Comment by manojnkumar** [comment]
> @manojnkumar what's the implication or making this change?
> I'm not ware of any sriov function testing on ppc64le or amd64, how it is verified?

The implication is that the operator becomes available in the UI (operatorhub).
The validation on ppc64le is done downstream.

File: manifests/4.8/sriov-network-operator.v4.8.0.clusterserviceversion.yaml (hunk 1 above)
Comment (manojnkumar) [reply manifests/4.8/sriov-network-operator.v4.8.0.clusterserviceversion.yaml:374]: > There are several operands managed by SR-IOV Operator, do you think we need to label them with the same tags?(I'm not very famiilar with how multi-arch works, any reference would be helpful here).

It is only required for the operator, and not all the operands.

**Comment by zshi-redhat** [comment]
> > @manojnkumar what's the implication or making this change?
> > I'm not ware of any sriov function testing on ppc64le or amd64, how it is verified?
> 
> The implication is that the operator becomes available in the UI (operatorhub).
> The validation on ppc64le is done downstream.

Does the validation contain testing on actual sriov hardware? or does it only validate the deployment of this operator?

**Comment by manojnkumar** [comment]
> > > @manojnkumar what's the implication or making this change?
> > > I'm not ware of any sriov function testing on ppc64le or amd64, how it is verified?
> > 
> > 
> > The implication is that the operator becomes available in the UI (operatorhub).
> > The validation on ppc64le is done downstream.
> 
> Does the validation contain testing on actual sriov hardware? or does it only validate the deployment of this operator?

Yes, on actual sriov hardware (Mellanox ConnectX family).

**Comment by zshi-redhat** [comment]
@manojnkumar  It is required to have a linked BZ for this PR to merge since it has passed 4.8 feature freeze date. 
Do you have the bugzilla already created?

**Comment by manojnkumar** [comment]
> @manojnkumar It is required to have a linked BZ for this PR to merge since it has passed 4.8 feature freeze date.
> Do you have the bugzilla already created?

Added the reference.  Waiting for the target to be set to 4.8.0.

**Review by yselkowitz — COMMENTED** [review]
Aren't these supposed to be under manifest.labels, not spec.labels?

**Pull request merged**